src/logger.py

```
@@ -2,7 +2,7 @@
 import logging
 import os
 from datetime import datetime
-from typing import Any
+from typing import Mapping
```

Ruff (0.14.10)

Import from `collections.abc` instead: `Mapping`

Import from `collections.abc`

(UP035)